src/z3c/form/tests/test_bugfix.py

```diff
@@ -62,5 +62,90 @@ class TestForm(z3c.form.form.BaseForm):
         self.assertEqual({'text': 'a'}, content)
 
 
+class Mock(object):
+    pass
+
+
+class MockFormatter(object):
+    def format(self, value):
+        if value is None:
+            # execution should never get here
+            raise ValueError('Cannot format None')
+        return str(value)
+
+class MockLocale(object):
+    def getFormatter(self, category):
+        return MockFormatter()
+
+
+class OurNone(object):
+    # this quacks like None, but is NOT None
+    def __cmp__(self, other):
```

Comment:
This is not used in Python 3.
Even in Python 2 it should be unused because there is also `__eq__`.

Reply:
See https://coveralls.io/builds/21930709/source?filename=src/z3c/form/tests/test_bugfix.py#L81 for not used methods of this class.